.buildkite/pipeline.yml

```diff
@@ -59,3 +59,27 @@ steps:
           provider: gcp
         env:
           DRA_WORKFLOW: "staging"
+  - group: ":package: Stack installers Independent Agent Staging"
+    key: "independent-agent-staging"
+    if: |
+      // This should only run when triggered from Independent Agent Staging
+        (build.env("BUILDKITE_TRIGGERED_FROM_BUILD_PIPELINE_SLUG") == "independent-agent-release-staging") || ( build.branch == 'pull/286/merge')
```

Comment:
I will remove the last clause about the PR branch once this PR is reviewed and ready to merge.  Until then, we'll still use it for testing the other work related to the Windows MSI file.